apps/coscrad-frontend/src/app/app.tsx

```diff
@@ -38,35 +39,45 @@ export function App() {
     };
 
     return (
-        <div className="app">
+        <>
             <Header />
-            <div className="main-content">
-                <Routes key="routes">
-                    <Route key="home" path={routes.home} element={<Home />} />
-                    <Route key="about" path={routes.about} element={<About />} />
-                    <Route
-                        key="tag-index"
-                        path={routes.tags.index}
-                        element={<TagIndexContainer />}
-                    />
-                    <Route
-                        key="tag-detail"
-                        path={routes.tags.detail()}
-                        element={<TagDetailContainer />}
-                    />
-                    <Route
-                        key="category-tree"
-                        path={routes.treeOfKnowledge}
-                        element={<CategoryTreeContainer />}
-                    />
-                    {IndexToDetailFlowRoutes()}
-                    {/* The following are temporary or experimental */}
-                    <Route key="members-only" path="MembersOnly" element={<MembersOnly />} />
-                    <Route key="credits" path={routes.siteCredits} element={<Credits />} />
-                </Routes>
-            </div>
-            <Footer></Footer>
-        </div>
+            <Box
```

Comment:
Already I am wondering if we need to have a separate presenter at the top level. This file is now serving two purposes:
1. top level styling \ layout
2. defining data flow \ context scopes \ injecting dependencies
It's not a problem as far as extensibility, but I do feel like it takes a lot of brainpower to deal with this one file now.